### PR TITLE
Adding redux, Removing zoom block and moving to ES6

### PR DESCRIPTION
**Pull request by avivbueno**

### CODE REVIEW

**Review by eladg — COMMENTED** [review]
looks good, not a big progress though. Writing you on messenger

**Review by eladg — CHANGES_REQUESTED** [review]
Redux should be handled on a different file

File: src/js/reducers.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+var store = Redux.createStore(bartData);
+
+function bartData(state = { apiData: {} }, action) {
+  switch (action.type) {
+    case "BART_DATA_UPDATE":
+      if (state.apiData === undefined) {
+        return state;
+      }
+      state.apiData = action.apiData;
```
Comment (eladg) [inline src/js/reducers.js]: This should be in a function

Comment (eladg) [reply src/js/reducers.js]: state is immutable, you should not update it in this way

File: src/js/reducers.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+      return state;
```
Comment (eladg) [inline src/js/reducers.js]: you should not return `state` but the actual state, something like:
```javascript
{ apiDate: payload.data } 
```

Comment (avivbueno) [reply src/js/reducers.js]: @eladg fixed this in the new method in the new file (data.js)

File: src/js/reducers.js (hunk 1 above)
Comment (avivbueno) [reply src/js/reducers.js]: @eladg Moved to a new file, take a look. (data.js)

**Pull request merged**